### ISSUE TITLE
VCS Compile Issue

### ISSUE THREAD

**Issue by PauloCarvalhoSP** [adit0131/APB_VIP#1]
Hi,

I did try run a simulation using the following command:

make vcscomp sim USE_SIMULATOR=vcs WAVES=fsdb

But I had the following compile error:

Parsing design file '../master_agent/apb_uvc_master_pkg.sv'

Error-[SE] Syntax error
  Following verilog source has syntax error :
  "../master_agent/apb_uvc_master_pkg.sv", 16: token is ';'
  package apb_uvc_master_pkg;

I'm using Synopsys VCS V-2023.12-SP2

Is there any missing argument into the command line or in the Makefile?

Thanks,

Paulo

**Comment by neelpambhar10** [comment]
Hi 
  
   Are you integrate this vip into your environment ?

**Comment by adit0131** [comment]
Yes it was integrated into my environment.

Aditya Mishra
ASIC Verification Engineer
Mobile: +919925953343
Email: ***@***.******@***.***>

[image]<https://www.scaledge.io/>

Scaledge India Pvt. Ltd.
INDIA | USA | UK | CANADA
www.scaledge.io<http://www.scaledge.io/>

[image]<https://www.linkedin.com/company/scaledge-technology>

Send by Mobile
________________________________

**Comment by PauloCarvalhoSP** [comment]
Yes, I'm trying to run first a test from your VIP using your makefile from run folder.

**Comment by PauloCarvalhoSP** [comment]
The issue reported here is using the Makefile available inside the run folder of this VIP.
